examples/src/main/scala/org/apache/spark/examples/sql/hive/SparkHiveExample.scala

```diff
@@ -104,6 +103,60 @@ object SparkHiveExample {
     // ...
     // $example off:spark_hive$
```

Comment:
Do you not want the code below to render in the docs as part of the example? maybe not, just checking if that's intentional.

Reply:
@srowen Thank you for valueable feedback review, I have added that so it can help other develoeprs.

Reply:
@srowen Can you please review this cc\ @holdenk @sameeragarwal

Reply:
@srowen I have updated DDL when storing data with parititoning in Hive.
cc\ @HyukjinKwon @mgaido91 @markgrover @markhamstra

Reply:
Why do you turn the example listing off then on again? just remove those two lines

Reply:
@srowen I mis-understood your first comment. I have reverted as suggested. Please check now